frontend/web/churchlink/src/features/webeditor/grid/DynamicPageV2RendererBuilder.tsx

```
@@ -578,6 +578,64 @@ const renderNode = (
         </div>
       );
     }
+    case 'contactInfo': {
```

Biome (2.1.2)

[error] Duplicate case label.

The first similar label is here:


(lint/suspicious/noDuplicateCase)